File: 08-Testing-CSharp/Starter/Testing.Tests/MyClassTests.cs
namespace Testing.Tests;

public class MyClassTests
{
    private readonly MyClass _testee;

    public MyClassTests()
    {
        _testee = new MyClass();
    }

    [Fact]
    public void DoLoop_LogsMessages()
    {
        _testee.DoLoop();
        
        Assert.Fail("Exercise 1: Verify all messages logged by MyClass.DoLoop()");
    }
}

File: 08-Testing-Java/Starter/test/testing/tests/MyClassTests.java
package testing.tests;

import static org.junit.jupiter.api.Assertions.fail;
import org.junit.jupiter.api.Test;
import testing.MyClass;

class MyClassTests {
    private final MyClass testee = new MyClass();

    @Test
    void doLoop_LogsMessages() {
        testee.doLoop();

        fail("Exercise 1: Verify all messages logged by MyClass.doLoop()");
    }

}
